Update Add DAG Utility Files (#1567)
astronomer/docs: astro/develop-project.md

## Add DAG utility files

> Use the `include` folder to store additional utilities required by your DAGs. For example, helper functions, templated SQL scripts, and custom operators. Storing additional utilities in the `include` folder lets you deploy code without an image restart. 

Reviewer: ```suggestion
Use the `include` folder to store additional utilities required by your DAGs. For example, helper functions, templated SQL scripts, and custom operators.
```

I'd actually just wipe this as the fact about the image restart has more to do with the dag-only functionality below

> When you use the `astro deploy -—dags` command, the `include` folder in the Astro project directory is not deployed with your DAGs and is instead built into the Docker image with your other project files.

Reviewer: ```suggestion
Astronomer recommends storing the `include` folder inside of the `dags` directory of your Astro project. If you use [DAG-only deploys](deploy-code.md#deploy-dags-only), this:

- Allows you to deploy DAG code without an image restart by running `astro deploy --dags`.
- Makes sure that your DAGs can access your utility files when you deploy them.

Here is how the recommended directory structure might appear:
```

I'm jumbling the sentences here @kwcanuck but bear with me! I just think let's start with the recommendation > benefits > example, and then end this section with the alternative

> If you are deploying only DAGs, Astronomer recommends moving the `include` folder into the `dags` directory so that your DAGs can access your utility files.  Here is how the recommended directory structure might appear:

Reviewer: ```suggestion
```

```bash
    .
    ├── airflow_settings.yaml
    ├── dags
    │   └── include
            └── helper_functions
                └── helper.py
            └── templated_SQL_scripts
    │       └── custom_operators
    ├── Dockerfile
    ├── tests
    │   └── test_dag_integrity.py
    ├── packages.txt
    ├── plugins
    │   └── example-plugin.py
    └── requirements.txt
```

> The Triggerer does not restart when you deploy only DAGs. See [Deploy DAGs only](deploy-code.md#deploy-dags-only).

Reviewer: ```suggestion
If you do not use DAG-only deploys or you decide to keep the `include` directory separate from the `dags` directory, the `include` folder in your Astro project directory is not deployed with your DAGs and is instead built into the Docker image with your other project files. See [Deploy DAGs only](deploy-code.md#deploy-dags-only).
```

Triggerer was a relevant question but I think less for this doc and more for the deploy code doc itself

Use the `include` folder to store additional utilities required by your DAGs. For example, helper functions, templated SQL scripts, and custom operators.

Astronomer recommends storing the `include` folder inside the `dags` directory of your Astro project. If you're using [DAG-only deploys](deploy-code.md#deploy-dags-only), this allows you to deploy DAG code without an image restart by running `astro deploy --dags` and it makes sure that your DAGs can access your utility files when you deploy them.

Here is how the recommended directory structure might appear:

```bash
    .
    ├── airflow_settings.yaml
    ├── dags
    │   └── include
            └── helper_functions
                └── helper.py
            └── templated_SQL_scripts
    │       └── custom_operators
    ├── Dockerfile
    ├── tests
    │   └── test_dag_integrity.py
    ├── packages.txt
    ├── plugins
    │   └── example-plugin.py
    └── requirements.txt
```

> If you do not use DAG-only deploys or you decide to keep the `include` directory separate from the `dags` directory, the `include` folder in your Astro project directory is not deployed with your DAGs and is built into the Docker image with your other project files. See [Deploy DAGs only](deploy-code.md#deploy-dags-only).

Author: ```suggestion
If you do not use DAG-only deploys or you decide to keep the `include` directory separate from the `dags` directory, the `include` folder in your Astro project directory is not deployed with your DAGs and is built into the Docker image with your other project files. 
```
